bcda/main_test.go

```diff
@@ -80,42 +80,42 @@ func (s *MainTestSuite) TestCreateACO() {
 	// Blank UUID
 	badUserUUID, err := createUser("", name, email)
 	assert.NotNil(s.T(), err)
-	assert.Nil(s.T(), badUserUUID)
+	assert.Equal(s.T(), badUserUUID, "")
```

Comment:
Would it be possible to swap the order on these `assert.Equal()` calls to have the expected value come before actual? https://github.com/stretchr/testify/blob/master/assert/assertions.go#L328

Reply:
Done!